website/static/_redirects

```diff
@@ -32,7 +32,6 @@
 /docs/new-architecture-library-android                     https://github.com/reactwg/react-native-new-architecture#guides
 /docs/new-architecture-library-ios                         https://github.com/reactwg/react-native-new-architecture#guides
 /docs/new-architecture-app-intro                           https://github.com/reactwg/react-native-new-architecture#guides
-/docs/react-18-and-react-native                            https://github.com/reactwg/react-native-new-architecture#guides
```

Comment:
Maybe this should redirect to /docs/0.69/react-18-and...

Reply:
Yup good point